src/legacy/core_plugins/metrics/public/components/annotations_editor.js

```diff
@@ -148,16 +176,27 @@ class AnnotationsEditor extends Component {
               <EuiFlexItem>
                 <EuiFormRow
                   id={htmlId('queryString')}
-                  label={(<FormattedMessage
-                    id="tsvb.annotationsEditor.queryStringLabel"
-                    defaultMessage="Query string"
-                  />)}
+                  label={
+                    <FormattedMessage
+                      id="tsvb.annotationsEditor.queryStringLabel"
+                      defaultMessage="Query string"
+                    />
+                  }
                   fullWidth
                 >
-                  <EuiFieldText
-                    onChange={this.handleChange(model, 'query_string')}
-                    value={model.query_string}
-                    fullWidth
+                  <QueryBar
```

Comment:
@TinaHeiligers  @Bargs Can we disable the refresh functionality for the QueryBar? 

With this one we have 4 components which somehow related to the refresh functionality. Adding one more refresh button can confuse our users. 
Also it's not working property with Auto-update. I expect that if user turn on this check-box next filter should be applying without pressing the refresh/update button.

![image](https://user-images.githubusercontent.com/20072247/57567472-830d4980-73e2-11e9-942b-93c4b4ff51c3.png)

Reply:
Yep, that's already in progress.